packages/cdk/lib/constructs/batch.ts

```diff
@@ -168,12 +175,17 @@ export class Batch extends Construct {
 
   private renderComputeEnvironment(options: ComputeOptions): IComputeEnvironment {
     const computeType = options.computeType || defaultComputeType;
+    const subnets = {
+      // Even if we use public subnets, CDK will assign security groups that don't allow inbound
```

Comment:
This comment is currently accurate but might be misleading because it is only true if the security groups are not changed in future updates to this stack.